docs/send-data/opentelemetry-collector/data-source-configurations/mapping-records-resources.md

```diff
@@ -0,0 +1,78 @@
+---
+id: mapping-records-resources
+title: Mapping OpenTelemetry Concepts to Sumo Logic
+sidebar_label: Mapping OTel Concepts to Sumo Logic
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This page describes how to map your OpenTelemetry attributes like Resources, Records, and Instrumentation Scope, to Sumo Logic concepts like Fields.
+
+OpenTelemetry has a [rich data model](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto) that is internally constructed out of several layers. For all signals, these can be broken down into following:
+
+* **Resource**. Includes attributes describing the resource from which given set of data comes from. Should follow [resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions).
```

Comment:
@swiatekm-sumo All of the links in this doc that end in `/semantic_conventions` are broken. Do you recommend linking to the parent folder (in this case, https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource), or directly to the semantic convention info (which is https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#semantic-attributes-with-sdk-provided-default-value)?

Reply:
I'd say directly to the semantic conventions repo https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#semantic-attributes-with-sdk-provided-default-value.

The semantic conventions have been moved some time ago from the [opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/) repository to its own repository [semantic-conventions](https://github.com/open-telemetry/semantic-conventions), so I definitely recommend to link to the new repo.